### ISSUE TITLE
[Bug]: INAV Logs detected but no gyro data extracted

### ISSUE THREAD

**Issue by b14ckyy** [gyroflow/gyroflow#68]
### Gyroflow version

v1.0.0-rc1

### What operating system are you using?

Windows 11 - 22000.438

### What GPU are you using?

Core i7 7700K

### What happened?

INAV Blackbox Logs (TXT format) are detected and INAV version is shown but no gyro traces appear and no sync possible. 
decoded csv files are not accepted at all. 
Test log: 
[LOG00044.TXT](https://github.com/gyroflow/gyroflow/files/8006350/LOG00044.TXT)

![image](https://user-images.githubusercontent.com/33039058/152606212-de10ae3b-7936-484c-aaa4-59cebb46d198.png)

**Comment by Nemeton** [comment]
Same on Mac mini M1

**Comment by Fersy2003** [comment]
same on win11 with inav 3.0.1

[LOG00004.TXT](https://github.com/gyroflow/gyroflow/files/8008503/LOG00004.TXT)

**Comment by lukdut** [comment]
The save issue with inav 4.0.0. I tried to convert my *.txt log file into *.csv using blackbox-tools-4.0.1_win32, I got 450+Mb of gyro data, but it doesn't work neither with the error "Unsupported file format". The save *.csv output file works just fine with gyroflow-0.3.0-beta-win64 btw.

**Comment by AlexZaiets** [comment]
same on windows 10 - 21H2(19044.1469), Gyroflow v1.0.0-rc1, INAV - 4.0.0
[LOG00037.TXT](https://github.com/gyroflow/gyroflow/files/8018381/LOG00037.TXT)

**Comment by Fersy2003** [comment]
nope, doesn't works with inav 3.0.1 files

[LOG00004.TXT](https://github.com/gyroflow/gyroflow/files/8018598/LOG00004.TXT)

**Comment by b14ckyy** [comment]
> nope, doesn't works with inav 3.0.1 files
> 
> [LOG00004.TXT](https://github.com/gyroflow/gyroflow/files/8018598/LOG00004.TXT)

Use Blackbox tools to decode into a CSV file and it works: 
![image](https://user-images.githubusercontent.com/33039058/152876174-d4336bdf-925d-492c-b5d4-65e2fea1132c.png)

**Comment by Fersy2003** [comment]
> Use Blackbox tools to decode into a CSV file and it works: !

> > nope, doesn't works with inav 3.0.1 files
> > [LOG00004.TXT](https://github.com/gyroflow/gyroflow/files/8018598/LOG00004.TXT)
> 
> Use Blackbox tools to decode into a CSV file and it works: ![image](https://user-images.githubusercontent.com/33039058/152876174-d4336bdf-925d-492c-b5d4-65e2fea1132c.png)

you mean converted with the blackbox_decode.exe?

**Comment by b14ckyy** [comment]
@Fersy2003 yes with that one decoded, just use the csv file and it works now.

**Comment by Fersy2003** [comment]
> @Fersy2003 yes with that one decoded, just use the csv file and it works now.

unfortunately it does not work for m
![Unbenannt](https://user-images.githubusercontent.com/99052626/152950260-af706998-f9eb-4ae1-af91-c12345b8c994.jpg)
e

**Comment by b14ckyy** [comment]
@Fersy2003 you have to use the latest development build that @avsaase has linked above.

**Comment by Fersy2003** [comment]
> @Fersy2003 you have to use the latest development build that @avsaase has linked above.

ja, läuft nicht..

[LOG00004.01.zip](https://github.com/gyroflow/gyroflow/files/8023300/LOG00004.01.zip)
.

**Comment by b14ckyy** [comment]
> > @Fersy2003 you have to use the latest development build that @avsaase has linked above.
> 
> ja, läuft nicht..
> 
> [LOG00004.01.zip](https://github.com/gyroflow/gyroflow/files/8023300/LOG00004.01.zip) .

What Blackbox Tools version do you use? With the latest one I see the same error. With the 0.4.7-alpha0 version it accepts the CSV but no traces now with the latest GF build. Need to check with the devs.

**Comment by Fersy2003** [comment]
> > > @Fersy2003 you have to use the latest development build that @avsaase has linked above.
> > 
> > 
> > ja, läuft nicht..
> > [LOG00004.01.zip](https://github.com/gyroflow/gyroflow/files/8023300/LOG00004.01.zip) .
> 
> What Blackbox Tools version do you use? With the latest one I see the same error. With the 0.4.7-alpha0 version it accepts the CSV but no traces now with the latest GF build. Need to check with the devs.

ahh ok, that explains it

**Comment by ilya-epifanov** [comment]
The issue is fixed, no new reports are needed. Please wait for an update.

**Comment by AdrianEddy** [comment]
Fixed in e532d34025823dacbeaf2e8fb90708b3df56a90c
Download here: https://github.com/gyroflow/gyroflow/suites/5220852577/artifacts/160018759

**Comment by GarryBu** [comment]
Windows11
1-4RC
Inav 4.0.0

https://user-images.githubusercontent.com/60334441/155184772-0a32d0d8-d351-45d1-adae-c76f25420275.mp4

[LOG00007.zip](https://github.com/gyroflow/gyroflow/files/8118717/LOG00007.zip)
